tracer/src/Datadog.Trace/RemoteConfigurationManagement/RemoteConfigurationManager.cs

```diff
@@ -78,29 +73,18 @@ internal class RemoteConfigurationManager : IRemoteConfigurationManager
             discoveryService.SubscribeToChanges(SetRcmEnabled);
         }
 
-        public static RemoteConfigurationManager? Instance { get; private set; }
-
         public static RemoteConfigurationManager Create(IDiscoveryService discoveryService, IRemoteConfigurationApi remoteConfigurationApi, RemoteConfigurationSettings settings, string serviceName, ImmutableTracerSettings tracerSettings, IGitMetadataTagsProvider gitMetadataTagsProvider, IRcmSubscriptionManager subscriptionManager)
```

Comment:
Arguably we can delete this Create method entirely and just use the constructor 😉

Reply:
This is left as an exercise for the reader